src/webui/frontend/src/routes/Dashboard/Connectors.tsx

```diff
@@ -146,36 +146,36 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
                       ))}
                     </Select>
                   ) : (
-                    React.createElement(opt.secret || opt.mask_on_screen ? Input.Password : Input, {
-                      name: opt.name,
-                      defaultValue: opt.default,
-                      value: optionValues[opt.name],
-                      addonBefore: opt.prefix,
-                      addonAfter: opt.postfix,
-                      placeholder: opt.placeholder,
-                      autoComplete: 'off',
-                      onBlur: (e: any) => {
-                        if (opt.required && opt.default && e.target.value === '') {
-                          this.changeOption(opt.name, opt.default);
-                        }
-                      },
-                      onChange: (e: any) => {
-                        // todo why doesn't ref to e work here w/ prevState?
-                        this.changeOption(opt.name, e.target.value);
-                      },
-                    })
-                  )}
+                      React.createElement(opt.secret || opt.mask_on_screen ? Input.Password : Input, {
```

Comment:
not sure whether this is text editor or accident but let's keep consistent indent, i.e. 2-space indent in this file

Reply:
The default autoformatter seems to be using a different style guide. I switched the formatter and now it seems to do 2-space indent for that particular construct.